### PR TITLE
Fetch and render the github document if the user opened petal from EDIT ME link

### PR DESCRIPTION
**Pull request by marmoure**
Fetch the raw document from GitHub then transform it into a prosemirror state json.

# How to test this PR
* Checkout the branch `git checkout feature/render`
* Install dependencies `yarn install`
* Start the demo `yarn run start:demo`
* Make a request to `http://localhost:1234/?ghrepo=evolvedbinary/prosemirror-lwdita&source=packages/prosemirror-lwdita-demo/example-xdita/02-short-file.xml&referer=htwsrhtshrts`
* Authenticate Petal Github APP 
* `02-short-file.xml` should be rendered on the editor
image

### CODE REVIEW

File: packages/prosemirror-lwdita-demo/src/example.ts [hunk 1]
```diff
@@ -6,19 +6,25 @@ import jsonDocLoader from "./doc";
...
-// TODO(YB): fetch and render the file from the URL
+const urlParams = processRequest() as { key: string, value: string }[];
 
 // if the user passes a file in the URL, load that file
 // otherwise load the default file
-const loadJsonDoc = jsonDocLoader;
+let loadJsonDoc = jsonDocLoader;
+if(urlParams) {
+  const ghrepo = urlParams.find((param) => param.key === 'ghrepo');
+  const source = urlParams.find((param) => param.key === 'source');
```
Comment (plutonik-a) [inline packages/prosemirror-lwdita-demo/src/example.ts:23]: This logic should be moved into `request.ts` in `prosemirror-lwdita`. 
We shouldn't process the parameters here but instead, retrieve the values from the request module.
Plus, `ghrepo` and `source` are defined in a constant there and we expect to maintain them only there in one single place. Every update of the parameter names would require to also update them in this "demo" package.

**Review by plutonik-a — CHANGES_REQUESTED** [review]
@marmoure Works great!
I have only a little request for moving the handling of the parameters to the appropriate package & module.

File: packages/prosemirror-lwdita-demo/src/example.ts (hunk 1 above)
Comment (marmoure) [reply packages/prosemirror-lwdita-demo/src/example.ts:23]: The way `urlParqams` variable is structured require us to use the array find method, and we need to provide a static key.
As of now I don't see another way of getting `ghrepo`, `source`.

**Review by plutonik-a — APPROVED** [review]

File: packages/prosemirror-lwdita-demo/src/example.ts (hunk 1 above)
Comment (plutonik-a) [reply packages/prosemirror-lwdita-demo/src/example.ts]: Fixed with https://github.com/evolvedbinary/prosemirror-lwdita/pull/394/commits/f7f7ee911aeeb7e35718c54846a8daafefa19477

**Pull request merged**